BiometricLockerTests/BiometricLockerTests.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  BiometricLockerTests.swift
+//  BiometricLockerTests
+//
+//  Created by Igor Ranieri on 16.04.18.
+//  Copyright © 2018 Bakken&Bæck. All rights reserved.
+//
+
+import XCTest
+import LocalAuthentication
+@testable import BiometricLocker
+
+class BiometricLockerTests: XCTestCase {
+
+    let locker = BiometricLocker()
+
+    override func setUp() {
+        super.setUp()
+        self.locker.unlock()
+        self.locker.unlockedTimeAllowance = LATouchIDAuthenticationMaximumAllowableReuseDuration
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func testLockUnlock() {
+        XCTAssertFalse(self.locker.isLocked)
+
+        self.locker.lock()
+        XCTAssertFalse(self.locker.isLocked)
```

Comment:
I would actually break this out into testing immediate vs non-immediate lock and unlock

Reply:
Done!